D1371R3.md

```diff
@@ -371,8 +370,8 @@ Op parseOp(Parser& parser) {
 ## Basic Syntax
 
 > | `inspect constexpr`*~opt~* `(` *init-statement~opt~* *condition* `)` *trailing-return-type~opt~* `{`
-> |     *pattern* *guard~opt~* `=>` *expression* `,`
-> |     *pattern* *guard~opt~* `:` *statement*
+> |     *pattern* *guard~opt~* `:` *expression* `,`
+> |     *pattern* *guard~opt~* `:` `{` *statement-seq* `}`
```

Comment:
Should we include  `!~opt~` here? So that we can later refer to it in the Basic Model.

Reply:
Sure, I'm happy to introduce it. I was finding it hard to break down the work into reviewable chunks but how about I mention it here and I/we expand on it in a future PR?